sherpa/ui/utils.py

```diff
@@ -332,19 +333,85 @@ def reduce_ufunc(func):
 
 
 class ModelWrapper(NoNewAttributesAfterInit):
+    """Wrap up a model class so we can create instances easily.
+
+    The created instances can be used in a "model language" to create
+    complex expressions. If mdl1 and mdl2 have been created by
+    ModelWrapper then a user can say mdl1.n1 + mdl2.n2 to create model
+    instances named n1 and n2 and then return the expression which
+    represents their sum. Note that the model component is stored in
+    the session object.
+
+    """
+
+    def __init__(self, session, modeltype, args=(), kwargs=None):
+        # This is an internal class so do not bother with
+        # sherpa.utils.err exceptions.
+        #
+        if not isinstance(session, Session):
+            raise ValueError("session is not a Session instance")
```

Comment:
```suggestion
            raise ValueError(f"{session} is not a Session instance")
```
Just a suggestion...

Reply:
So, I think the message could be

    The session argument is not a Session instance
    f"session={session} is not a Session instance

as I wanted to easily identify which argument is wrong. However, I don't see this really ever getting hit, as this is all internal code, so I'm not sure it's worth the effort. Thoughts?

Reply:
I should also point out that for this, and the following check, I could not be bothered trying to find a sherpa error class to use, and just went with the normal Python exceptions, as I really don't expect these to ever occur.

There is a discussion we could have about the sherpa error class, but that is a significantly-larger issue that we don't want to touch here.

Reply:
I agree it's not worth the effort; just a suggestion.